master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -287,6 +283,8 @@ spec:
       # it isn't governed by policy that would prevent it from working.
       hostNetwork: true
       serviceAccountName: calico-kube-controllers
+      tolerations:
+        - operator: "Exists"
```

Comment:
Do we need "effect: NoSchedule" as well?

I think we probably should specify that.